python/cudf/cudf/tests/test_parquet.py

```diff
@@ -2845,13 +2845,32 @@ def test_to_parquet_row_group_size(
         fname, row_group_size_bytes=size_bytes, row_group_size_rows=size_rows
     )
 
-    num_rows, row_groups, col_names = cudf.io.read_parquet_metadata(fname)
+    num_rows, row_groups, col_names, row_group_metadata = cudf.io.read_parquet_metadata(fname)
```

Comment:
```suggestion
    num_rows, row_groups, col_names, _ = cudf.io.read_parquet_metadata(fname)
```

We could do this, because we are anyway not utilizing `row_group_metadata` anywhere here.